src/Build/Evaluation/EvaluatorMetadataTable.cs

```diff
@@ -43,18 +43,7 @@ public EvaluatorMetadataTable(string implicitItemType)
         /// <summary>
         /// Enumerator over the entries in this table
         /// </summary>
-        internal IEnumerable<EvaluatorMetadata> Entries
-        {
-            get
-            {
-                if (_metadata == null)
-                {
-                    return ReadOnlyEmptyList<EvaluatorMetadata>.Instance;
-                }
-
-                return _metadata.Values;
-            }
-        }
+        internal IEnumerable<EvaluatorMetadata> Entries => _metadata?.Values ?? (IEnumerable<EvaluatorMetadata>) Array.Empty<EvaluatorMetadata>();
```

Comment:
I sure can! I thought I tried something like that and this was required but I was Doing It Wrong.